core/register_command.php

```diff
@@ -187,6 +187,7 @@
 	$application->add(new OC\Core\Command\Security\ImportCertificate(\OC::$server->getCertificateManager(null)));
 	$application->add(new OC\Core\Command\Security\RemoveCertificate(\OC::$server->getCertificateManager(null)));
 	$application->add(new OC\Core\Command\Security\ResetBruteforceAttempts(\OC::$server->getBruteForceThrottler()));
+	$application->add(new OC\Core\Command\Security\ListRoutes(\OC::$server->getRouter()));
```

Comment:
Let's use auto-wiring:

```suggestion
	$application->add(\OC::$server->get(\OC\Core\Command\Security\ListRoutes::class));
```